server/data/prisonApiClient.test.ts

```diff
@@ -1,20 +1,25 @@
 import nock from 'nock'
 import config from '../config'
-import PrisonApiClient, { prisonApiClientBuilder } from './prisonApiClient'
+import PrisonApiClient from './prisonApiClient'
 import { InmateDetail, OffenderRestrictions, PagePrisonerBookingSummary, VisitBalances } from './prisonApiTypes'
 import TestData from '../routes/testutils/testData'
 
 describe('prisonApiClient', () => {
   let fakePrisonApi: nock.Scope
-  let client: PrisonApiClient
+  let prisonApiClient: PrisonApiClient
   const token = 'token-1'
 
   beforeEach(() => {
     fakePrisonApi = nock(config.apis.prison.url)
-    client = prisonApiClientBuilder(token)
+    prisonApiClient = new PrisonApiClient(token)
   })
 
   afterEach(() => {
+    if (!nock.isDone()) {
```

Comment:
Saw this `afterEach()` check on some other projects; looked useful so added to each API client test.